skype/skype-ps/skype/Set-CsTeamsAppPermissionPolicy.md

```diff
@@ -43,7 +43,65 @@ As an admin, you can use app permission policies to enable or block specific app
 ## EXAMPLES
 
 ### Example 1
-Intentionally omitted.
+
+No apps ae blocked,allow all Microsoft apps,Third-party apps,Custom apps.

```

Comment:
Please move the description below the code

Reply:
got it,thanks！